test/test-network/conf/bond99.network

```diff
@@ -0,0 +1,6 @@
+[Match]
+Name=bond99
+
+[Network]
+VLAN=vlan6
+IPv6AcceptRA=false
```

Comment:
we do we explicitly disable `IPv6AcceptRA` everywhere? Wouldn't it make more sense if there's saner defaults for enslaved interfaces?

Reply:
This is just for testing. If no RA received, then it takes ~12sec the link to be in 'configured' state. But we usually wait for 5sec in the testing script.

> Wouldn't it make more sense if there's saner defaults for enslaved interfaces?

I am not sure. Maybe so, but may not. I have no idea about that now.